src/main/java/edu/usf/cutr/grha/gtfs/TncToGtfsWriter.java

```diff
@@ -0,0 +1,127 @@
+package edu.usf.cutr.grha.gtfs;
+
+import edu.usf.cutr.grha.model.ChicagoTncData;
+import org.onebusaway.gtfs.model.*;
+import org.onebusaway.gtfs.model.calendar.ServiceDate;
+import org.onebusaway.gtfs.serialization.GtfsWriter;
+
+import java.io.File;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class TncToGtfsWriter {
+
+    public String filePath;
+
+    public TncToGtfsWriter(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void write(List<ChicagoTncData> chicagoTncDataList) {
+        GtfsWriter writer = new GtfsWriter();
+        writer.setOutputLocation(new File(filePath));
+
+        // Creates a single fake agency in agency.txt for TNC trips
+        Agency agency = newFakeAgency();
+        writer.handleEntity(agency);
+
+        // Creates a single fake route in routes.txt that all TNC trips can be assigned to
+        Route route = newFakeRoute(agency);
+        writer.handleEntity(route);
+
+        int counter = 0;
```

Comment:
Let's start the counter at `1` if it's used for IDs - having `0` for an ID isn't standard GTFS practice.